anaconda_anon_usage/tokens.py

```diff
@@ -24,6 +27,41 @@ def version_token():
     return __version__
 
 
+@cached
+def system_token():
+    """
+    Returns the system/organization token. Unlike the other
+    tokens, it is desirable for this token to be stored in
+    a read-only/system location, presumably installed
+    The system/organization token can be stored anywhere
+    in the standard conda search path. Ideally, an MDM system
+    would place it in a read-only system location.
+    """
+    # Do not import SEARCH_PATH directly since we need to
+    # temporarily patch it for testing
+    for path in c_constants.SEARCH_PATH:
+        # Only consider directories where
+        # .condarc could also be found
+        if not path.endswith("/.condarc"):
+            continue
+        parts = path.split("/")
+        if parts[0].startswith("$"):
+            parts[0] = environ.get(parts[0][1:])
+            if not parts[0]:
+                continue
+        parts[-1] = "org_token"
```

Comment:
```suggestion
        parts[-1] = "org_token"
```

Just noting that the pull request descriptions speak of `aau_token` and maybe it would be good to move this name into a constant

Reply:
Good catch. I had to switch the name because it's possible for this token to live in the same directory as `aau_token`